scripts/update-contributors.ts

```diff
@@ -1,23 +1,27 @@
 import { promises as fs } from 'fs'
 import fetch from 'node-fetch'
 
-const { GITHUB_TOKEN: token } = process.env
-
 interface Contributor {
   login: string
+  avatar_url?: string
 }
 
 async function fetchContributors() {
-  const collaborators: string[] = []
+  const collaborators: string[][] = []
   const res = await fetch('https://api.github.com/repos/vitest-dev/vitest/contributors', {
     method: 'get',
     headers: {
-      'authorization': `bearer ${token}`,
       'content-type': 'application/json',
     },
   })
   const data = await res.json() as Contributor[] || []
-  collaborators.push(...data.map(i => i.login))
+  collaborators.push(...data.map(({ login, avatar_url }) => {
+    // optimize the avatar size: check avatar and contributors components
```

Comment:
The size doesn't matter much, let's keep the codebase simple

Reply:
I don't think so, just compare these 2:

Your avatar using `https://github.com/antfu.png`

![imagen](https://user-images.githubusercontent.com/6311119/151466423-1c33dc4d-1d5b-4ad9-8691-debff4843a93.png)

And your avatar using `https://github.com/antfu.png?size=125` or `https://avatars.githubusercontent.com/u/11247099?v=4&s=125`:

![imagen](https://user-images.githubusercontent.com/6311119/151467166-6c8cf123-7c62-4492-a6fe-eb3c2835a982.png)

Reply:
744KB